File: RFPParser/Zbizlink.RFPServices/ViewModels/Insert/SummarySynonymBridges.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Zbizlink.RFPServices.ViewModels.Insert
{
   public class SummarySynonymBridges
    {
        public decimal UserId { get; set; }
        public decimal SummarySynonymId { get; set; }
        public bool Assign { get; set; }
        public List<BridgeSummarySynonymAgency> BridgeSummarySynonymAgencyList { get; set; }
        public List<BridgeSummarySynonymContractVehicle> BridgeSummarySynonymContractVehicleList { get; set; }
        public List<BridgeSummarySynonymIndustry> BridgeSummarySynonymIndustryList { get; set; }
        public List<BridgeSummarySynonymOpportunityType> BridgeSummarySynonymOpportunityTypeList { get; set; }
        public List<BridgeSummarySynonymStates> BridgeSummarySynonymStatesList { get; set; }
    }
}

File: Presentation/Zbizlink.ParserClient/ClientApp/src/app/CampaignManagement/models/summary-synonym-bridges.ts
import { BridgeSummarySynonymAgencyList } from './bridge-summary-synonym-agency-list';
import { BridgeSummarySynonymContractVehicleList } from './bridge-summary-synonym-contract-vehicle-list';
import { BridgeSummarySynonymIndustryList } from './bridge-summary-synonym-industry-list';
import { BridgeSummarySynonymOpportunityTypeList } from './bridge-summary-synonym-opportunity-type-list';
import { BridgeSummarySynonymStatesList } from './bridge-summary-synonym-states-list';

export class SummarySynonymBridges {
    // public decimal UserId { get; set; }
    //     public decimal SummarySynonymId { get; set; } 
    BridgeSummarySynonymAgencyList: Array<BridgeSummarySynonymAgencyList>;
    BridgeSummarySynonymContractVehicleList: Array<BridgeSummarySynonymContractVehicleList>;
    BridgeSummarySynonymIndustryList: Array<BridgeSummarySynonymIndustryList>;
    BridgeSummarySynonymOpportunityTypeList: Array<BridgeSummarySynonymOpportunityTypeList>;
    BridgeSummarySynonymStatesList: Array<BridgeSummarySynonymStatesList>;
}
